--- cpdb-libs-2.0b7/cpdb/Makefile.am.orig	2025-02-20 00:11:24.000000000 +0100
+++ cpdb-libs-2.0b7/cpdb/Makefile.am	2025-09-14 16:41:15.844096498 +0200
@@ -29,6 +29,7 @@ libcpdb_frontend_la_CPPFLAGS += $(GIOUNI
 libcpdb_frontend_la_CPPFLAGS += -I..
 
 libcpdb_frontend_la_LIBADD  = -lpthread -lm -lcrypt
+libcpdb_frontend_la_LIBADD  = libcpdb.la
 libcpdb_frontend_la_LIBADD += $(GLIB_LIBS)
 libcpdb_frontend_la_LIBADD += $(GIO_LIBS)
 libcpdb_frontend_la_LIBADD += $(GIOUNIX_LIBS)
